endpoints-framework/src/main/java/com/google/api/server/spi/request/ServletRequestParamReader.java

```diff
@@ -176,30 +177,40 @@
           params[i] = (node == null) ? null : objectReader.forType(clazz).readValue(node);
           logger.log(Level.FINE, "deserialize: {0} {1} injected into unnamed param[{2}]",
               new Object[]{clazz, params[i], i});
-        } else if (StandardParameters.isStandardParamName(name)) {
-          params[i] = getStandardParamValue(node, name);
         } else {
-          JsonNode nodeValue = node.get(name);
-          if (nodeValue == null) {
-            params[i] = null;
+          if (StandardParameters.isStandardParamName(name)) {
```

Comment:
Any reason to turn the `else if/else` into a nested `else { if/else }`?
It looks strictly equivalent, am I misreading?

Reply:
It was just so that the null check can exist in the newly created else block, rather than re-checking for parameter name existence. Everything else is equivalent.